gmso/utils/units.py

```diff
@@ -79,12 +76,352 @@ def default_reg():
             A unyt registry with commonly used conversions defined.
         """
         reg = u.UnitRegistry()
-        conversion = (
-            1 * getattr(u.physical_constants, "elementary_charge").value
-        )
-        dimensionsList = [u.dimensions.current_mks, u.dimensions.time]
-        dim = np.prod(dimensionsList)
-        name = "elementary_charge"
-        symbol = r"\rm{e}"
-        reg.add(name, conversion, dim, symbol)
+        register_general_units(reg)
         return reg
+
+
+@staticmethod
+def register_general_units(reg: u.UnitRegistry):
+    """Register units that are generally useful to a basic unyt UnitSystem."""
+    conversion = 1 * getattr(u.physical_constants, "elementary_charge").value
+    dim = u.dimensions.current_mks * u.dimensions.time
+    reg.add(
+        "elementary_charge",
+        conversion,
+        dim,
+        r"\rm{e}",
+    )  # proton charge
+    conversion = (
+        1 * getattr(u.physical_constants, "boltzmann_constant_mks").value
+    )
+    dim = u.dimensions.energy / u.dimensions.temperature
+    reg.add(
+        "kb", base_value=conversion, dimensions=dim, tex_repr=r"\rm{kb}"
+    )  # boltzmann temperature
+    conversion = (
+        4
+        * np.pi
+        * getattr(u.physical_constants, "reduced_planck_constant").value ** 2
+        * getattr(u.physical_constants, "eps_0").value
+        / (
+            getattr(u.physical_constants, "electron_charge").value ** 2
+            * getattr(u.physical_constants, "electron_mass").value
+        )
+    )
+    dim = u.dimensions.length
+    reg.add(
+        "a0", base_value=conversion, dimensions=dim, tex_repr=r"\rm{a0}"
+    )  # bohr radius
+    conversion = (
+        getattr(u.physical_constants, "reduced_planck_constant").value ** 2
+        / u.Unit("a0", registry=reg).base_value ** 2
+        / getattr(u.physical_constants, "electron_mass").value
+    )
+    dim = u.dimensions.energy
+    reg.add(
+        "Ehartree",
+        base_value=conversion,
+        dimensions=dim,
+        tex_repr=r"\rm{Ehartree}",
+    )  # Hartree energy
+    conversion = np.sqrt(
+        10**9 / (4 * np.pi * getattr(u.physical_constants, "eps_0").value)
+    )
+    dim = u.dimensions.charge
+    reg.add(
+        "Statcoulomb_charge",
+        base_value=conversion,
+        dimensions=dim,
+        tex_repr=r"\rm{Statcoulomb_charge}",
+    )  # Static charge
+
+
+class LAMMPS_UnitSystems:
+    """Set of a unit systems distributed in LAMMPS (https://docs.lammps.org/units.html)."""
+
+    def __init__(self, style: str, registry=None):
+        if registry:
+            self.reg_ = registry
+        else:
+            self.reg_ = GMSO_UnitRegistry().reg
+        self.system_ = self.usystem_from_str(styleStr=style, reg=self.reg_)
```

Comment:
Should we change it to `unit_system` to match with that of `Topology`, I just think `.system` might be a bit confusing.

Reply:
Addressed in [f21a6c2](https://github.com/mosdef-hub/gmso/pull/747/commits/f21a6c2b74d3bd172ccbc6d87ba3eec5773299a3)